### PR TITLE
docs(infraex): adds rest api option besides zbctl

### PR DESCRIPTION
**Pull request by Langleu**
## Description

InfraEx has several docs that use zbctl as an example. This PR adds mostly tabs and rest API as additional option.
Zbctl is deprecated but is still a valid use case for 8.6, especially for gRPC endpoints. Therefore, would keep it for minimum 8.6

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`). 8.6

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by mesellings** [comment]
@leiicamundi I can take a look at this for you - would next week be okay for you (hopefully Tuesday)?

**Comment by Langleu** [comment]
@mesellings sure, appreciate it 🙇

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 1]
```diff
@@ -54,7 +54,8 @@ Running dual-region setups requires the users to be able to detect any regional
   - In that guide, we're showcasing Kubernetes dual-region installation, based on the following tools:
     - [Helm (3.x)](https://helm.sh/docs/intro/install/) for installing and upgrading the [Camunda Helm chart](https://github.com/camunda/camunda-platform-helm).
     - [Kubectl (1.30.x)](https://kubernetes.io/docs/tasks/tools/#kubectl) to interact with the Kubernetes cluster.
-- [zbctl](./../../../apis-tools/cli-client/index.md) to interact with the Zeebe cluster.
+- (deprecated) [zbctl](./../../../apis-tools/cli-client/index.md) to interact with the Zeebe cluster.
```
Comment (akeller) [inline docs/self-managed/operational-guides/multi-region/dual-region-ops.md:57]: Quick comment here: Internally, the discussion has called this "deprecation," but externally, we are moving this to community support; see https://github.com/camunda/camunda-docs/pull/4228. I'm leaning toward removing this line and just promoting cURL.

Comment (akeller) [reply docs/self-managed/operational-guides/multi-region/dual-region-ops.md:57]: Thanks for the link! I wasn't sure if the blog went live with the deprecation terminology - still catching up from my recent FTO. I will resolve this.

**Comment by Langleu** [comment]
@leiicamundi agree, I added it for the missing output in the EKS Helm guide as well. We already use it everywhere else in dual-region and makes sense to align this.

**Comment by christinaausley** [comment]
@Langleu Looks like just a few conflicting files, CC @mesellings for review, or I'm happy to have a look! 😄

**Comment by akeller** [comment]
Removing `deploy` since it's currently broken. 😢

**Comment by akeller** [comment]
@Langleu should this be part of the 8.6 docs release? If so, please wrap up your changes so we can review and merge.

**Comment by akeller** [comment]
Merging per convo on Slack - https://camunda.slack.com/archives/C026U8GBNSW/p1728323652223659?thread_ts=1728319424.421399&cid=C026U8GBNSW

**Review by conceptualshark — APPROVED** [review]

**Pull request merged**